test/parallel/test-crypto-binary-default.js

```diff
@@ -46,11 +46,11 @@ assert.throws(function() {
 }, 'not enough data');
 
 // Test HMAC
-var h1 = crypto.createHmac('sha1', 'Node')
+const hmacHash = crypto.createHmac('sha1', 'Node')
                .update('some data')
                .update('to hmac')
                .digest('hex');
```

Comment:
A minor nit: the above 3 lines should probably be re-aligned with the `.` on line 49.